MekHQ/src/mekhq/campaign/mission/AtBDynamicScenarioFactory.java

```diff
@@ -2568,8 +2570,8 @@ public static void benchAllyUnit(UUID playerUnitID, String templateName, AtBDyna
                 }
             }
 
-            if ((botForce != null) && !botForce.getEntityList().isEmpty()) {
-                Entity swapTarget = botForce.getEntityList().get(0);
+            if ((botForce != null) && !botForce.getFixedEntityList().isEmpty()) {
+                Entity swapTarget = botForce.getFixedEntityList().get(0);
                 BenchedEntityData benchedEntity = new BenchedEntityData();
                 benchedEntity.entity = swapTarget;
                 benchedEntity.templateName = destinationTemplate.getForceName();
```

Comment:
This is the place I was unable to change to `getFullEntityList because this method does not include a campaign pointer anywhere and trying to add one is difficult given the number of places the function is called.